array.js

```diff
@@ -0,0 +1,11 @@
+const oldEmployees = ["Mike", "Nabil", "Cubby", "Mendes"];
+const newEmployees = ["Marcus", "Rasmus", "Matt"];
+const allEmployees = [];
```

Comment:
Hey! Instead of using a forEach loop to sum up the new and old employees into allEmployees you can using the spread operator instead. It works like this : 
const allEmployees ? [...oldEmployees, ...newEmployees]